.github/workflows/build-master.yml

```diff
@@ -8,7 +8,7 @@ on:
 jobs:
   build-static-page:
     name: build
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-18.04
```

Comment:
`ubuntu-latest` is 20.04? or is it 20.10?

What was the failure we saw?

Reply:
It was `Error: Process completed with exit code 255.` There was also a warning `Ubuntu-latest workflows will use Ubuntu-20.04 soon. For more details, see https://github.com/actions/virtual-environments/issues/1816`